docs/conf.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+#
+# Aldryn FAQ documentation build configuration file, created by
+# sphinx-quickstart on Wed Dec 10 15:42:58 2014.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# import sys
+import os
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = []
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['templates']
+
+# The suffix of source filenames.
+source_suffix = '.rst'
+
+# The encoding of source files.
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Aldryn FAQ'
+copyright = u'Divio AG'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
```

Comment:
I tried the same here as with: https://github.com/aldryn/aldryn-newsblog/blob/master/docs/conf.py#L53 using `aldryn_faq` while getting the following error:

```
Exception occurred:
  File "conf.py", line 53, in <module>
    import aldryn_faq
ImportError: No module named aldryn_faq
The full traceback has been saved in /var/folders/zm/m41tq5910_z7jr512n_dbxk80000gn/T/sphinx-err-5NEohp.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```